src/components/markdown-input.less

```diff
@@ -0,0 +1,74 @@
+@markdown-input-block-min-height: 100px;
+
+.MarkdownInput {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: 1fr;
```

Comment:
We'd need to let this guy take `width: 100%` of its parent, for cases like this:

<img width="637" alt="Screen Shot 2022-06-14 at 14 09 00" src="https://user-images.githubusercontent.com/8189923/173573784-90b8a185-f71a-4e20-9d75-5217d6a41bba.png">
👇 
<img width="618" alt="Screen Shot 2022-06-14 at 14 09 40" src="https://user-images.githubusercontent.com/8189923/173573802-18898757-3d2f-4658-93b7-a12c83f406dd.png">

Reply:
True, thanks, now it takes 100%